etl/src/main/java/org/pih/hivmigration/etl/sql/DeIdentifyMigrator.groovy

```diff
@@ -158,6 +158,64 @@ class DeIdentifyMigrator extends SqlMigrator {
                     WHEN p.source_infant_id % 19 = 18 THEN 'Lucas'
                 END)
             where given_name != 'UNKNOWN' and pn.person_id = p.person_id and (p.gender != 'F' or p.gender is null);
+            
+            
+            update person_name pn, person p
+            set pn.middle_name =
+                (CASE
+                    WHEN p.person_id % 19 = 0 THEN 'Charlotte'
```

Comment:
Let's do these middle names modulo 21 so that we get minimal repeats (21 is coprime to 20 and 19). You just have to pick two more names. Or go modulo 17 and drop two names. Either way.